### PR TITLE
chr

### PR DESCRIPTION
**Pull request by DogZeroX**
## About the PR 
Brings back the name "chr" with a randomMetadata component, it would be funnier if it supported percent chance.
**Screenshots**

unknown

**Changelog**

### CODE REVIEW

**Comment by Just-a-Unity-Dev** [comment]
chr

File: Resources/Prototypes/Datasets/Names/chair.yml [hunk 1]
```diff
@@ -0,0 +1,4 @@
+- type: dataset
+  id: chr
+  values:
+  - chr
```
Comment (RemieRichards) [inline Resources/Prototypes/Datasets/Names/chair.yml:4]: You need to actually add 'chair' to this, otherwise it'll always be named 'chr'

**Comment by PaulRitter** [comment]
cant wait for the typo bugreport

**Comment by DogZeroX** [comment]
https://tenor.com/view/you-wo-joker-joaqu-gif-23924191

**Comment by Just-a-Unity-Dev** [comment]
> Lame.

myctai, just because your issue bloat got closed doesnt mean that this PR is lame

**Comment by asperger-sind** [comment]
> I played plenty of ss13 and I _indeed don't get it_. Just think what this makes new players think. Just think what you really accomplished with this.

chr

**Comment by Just-a-Unity-Dev** [comment]
> I played plenty of ss13 and I _indeed don't get it_. Just think what this makes new players think. Just think what you really accomplished with this.

i'm _sure_ you're fun to be around, myctai

**Comment by metalgearsloth** [comment]
Be civil.

**Pull request merged**